recipes/rmats2sashimiplot/meta.yaml

```diff
@@ -20,5 +20,16 @@ build:
   noarch: python
   preserve_egg_dir: True
 
+test:
+  commands:
+    - rmats2sashimiplot -h
+  imports:
+    - pysam
```

Comment:
The tests for required packages are not needed, they're handled within the respective package